src/argtypes.c

```diff
@@ -307,7 +307,7 @@ TypeTuple *toArgTypes(Type *t)
         void visit(TypeStruct *t)
         {
             //printf("TypeStruct::toArgTypes() %s\n", t->toChars());
-            if (!t->sym->isPOD())
+            if (!t->sym->isPOD() || t->sym->fields.dim == 0)
```

Comment:
It probably should, although passing D types to C code is really in implementation defined territory.  Beyond that we don't have a stable ABI.